core/src/test/scala/org/apache/spark/JobCancellationSuite.scala

```diff
@@ -455,6 +464,8 @@ class JobCancellationSuite extends SparkFunSuite with Matchers with BeforeAndAft
     sc.cancelJobGroup("jobA")
     val e = intercept[SparkException] { ThreadUtils.awaitResult(jobA, 15.seconds) }.getCause
     assert(e.getMessage contains "cancel")
+    semExec.acquire(2)
+    assert(execLossReason == Seq("Command exited with code 53", "Command exited with code 53"))
```

Comment:
We could use the `ExecutorExitCode` constant here to make find usages work a little clearer.

Reply:
Sure. Will do.